examples/index.js

```diff
@@ -0,0 +1,113 @@
+/*
+ * Copyright 2015-2016 Red Hat, Inc. and/or its affiliates
+ * and other contributors as indicated by the @author tags.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+'use strict';
+
+const http = require('http');
+const dispatcher = require('httpdispatcher');
+
+const opentracing = require('opentracing');
+const hawkularAPM = require('../index');
+
+const SERVER_PORT = 9000;
+
+opentracing.initGlobalTracer(new hawkularAPM.APMTracer({
+    recorder: new hawkularAPM.ConsoleRecorder(),
+    sampler: new hawkularAPM.AlwaysSampledSampler(),
+}));
+
+// /hello handler
+dispatcher.onGet('/hello', function(req, res) {
+    const serverSpan = opentracing.globalTracer().startSpan('hello', {
+        tags: {
+            'span.kind': 'serer',
+            'http.status_code': 200,
+        }
+    });
+    res.writeHead(200);
+    res.end('Hello from Node.js!');
+    serverSpan.finish();
+});
+
+// /a handler
+dispatcher.onGet("/a", function(req, res) {
+    const serverSpan = opentracing.globalTracer().startSpan('/a');
+    serverSpan.setTag('span.kind', 'server');
+
+    const clientSpan = opentracing.globalTracer().startSpan('/b', {
+        childOf: serverSpan ,
+        tags: {
+            'span.kind': 'client',
+        }
+    });
+    http.get({
+        host: 'localhost',
+        port: SERVER_PORT,
+        path: '/b',
+        headers: createCarrier(opentracing.globalTracer(), clientSpan),
+    }, function (response) {
+        clientSpan.finish();
+    });
+
+    res.writeHead(200);
+    res.end('/a');
+    serverSpan.finish();
+});
+
+// /b handler
+dispatcher.onGet('/b', function(req, res) {
+    // TODO log something and report to the server
+    const spanContext = extractSpanContext(opentracing.globalTracer(), req.headers);
+    const serverSpan = opentracing.globalTracer().startSpan('/b', {
```

Comment:
I think it will be better to use another operation name - possibly GET or a business relevant name, and then put the URI/path in a tag with "http.url" name - i.e. the standard opentracing tag name.
Same comment on the client and server spans in the code above.